app/controllers/signup_controller.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from app import db
+from app.models import User
+
+from flask import jsonify, request
+
+from app.session_util import create_session
+
+
+class SignupController():
+    def handle(self):
+        data = json.loads(request.data)
```

Comment:
I think `request.json` will work if the front-end is sending the data back w/ `application/json`

Reply:
😮 oooh awesome!!!